src/sentry/snuba/outcomes.py

```diff
@@ -162,6 +163,21 @@ def map_row(self, row: MutableMapping[str, Any]) -> None:
             row["outcome"] = Outcome(row["outcome"]).api_name()
 
 
+class KeyDimension(Dimension):
+    def resolve_filter(self, raw_filter: Sequence[str]) -> List[int]:
+        def _parse_value(key_id: str) -> Outcome:
+            try:
+                return int(key_id)
```

Comment:
@markstory maybe this
```suggestion
                return Outcome(int(key_id))
```